### DISCUSSION TITLE
the next discussion

### DISCUSSION THREAD

**Discussion by WinTestEdt** [WinTestEdt/WinTestEdt#10] [1.1 General Help and Discussion]
here we have another discussion

**Comment by anton-vrba** [comment]
Here I reply with an email and no attachmnents

**Comment by WinTestEdt** [comment]
let me say some more

**Comment by anton-vrba** [comment]
and I reply with an email with a pdf attchment

**Reply by WinTestEdt** [reply]
email attachments get lost, they can only be included when using the web interface
[US-LEI-Tech-Notes(4).pdf](https://github.com/WinTestEdt/WinTestEdt/files/14070268/US-LEI-Tech-Notes.4.pdf)

**Reply by Alex-Simonic** [reply]
Using the web interface may be the preferred way. I will include an interface to the discussions in WinEdt's Help Menu (ShellExecute).

**Comment by anton-vrba** [comment]
replying to a reply with a pasted picture

**Reply by WinTestEdt** [reply]
pasted picture also got lost

**Comment by anton-vrba** [comment]
and only a text reply by email
